content/source/docs/cloud/api/modules.html.md

```diff
@@ -37,13 +37,19 @@ The Terraform Cloud Module Registry implements the [Registry standard API](../..
 - Get a specific module
 - Download the latest version of a module
 
+For publicly curated modules, the Terraform Cloud Module Registry acts as a proxy to the [Terraform Registry](https://registry.terraform.io) for the following:
+- List available versions for a specific module
```

Comment:
There should be a newline here to start the list:

```suggestion

- List available versions for a specific module
```

Reply:
Fixed